src/main/java/io/fabric8/vertx/maven/plugin/mojos/RunMojo.java

```diff
@@ -33,8 +36,111 @@
     requiresDependencyResolution = ResolutionScope.COMPILE_PLUS_RUNTIME)
 public class RunMojo extends AbstractRunMojo {
 
+    /**
```

Comment:
I had that doubt, when using CustomLauncher its the CustomerLauncher's responsibility to pass the needed run arguments right ?

What I mean is if I do use MyLauncher and pass the run arguments then the custom launcher has to pass it back to Vert.x Launcher.

Technically we should add these run arguments only if VERTX_LAUNCHER. That wha you mean here right?